src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,198 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final int OFFSET_ONE = 1;
+    private static final int EMPTY_LIST = 0;
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
+    private static class Node<E> {
```

Comment:
Don't use getters and setters in class Node. You have access to private fields of the inner class, so using getters or setters is redundant. Currently, there are no getters or setters, which is correct.